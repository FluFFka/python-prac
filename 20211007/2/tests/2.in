10.123456789, 9.78451204
