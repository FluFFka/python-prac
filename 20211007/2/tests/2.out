0.33894474900000127
